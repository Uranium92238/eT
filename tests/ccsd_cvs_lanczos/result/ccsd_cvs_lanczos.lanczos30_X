Nbr.      energy (Re) [eV]      energy (Im) [eV]     energy (Re) [a.u]    energy (Im) [a.u.]   Osc. strength
   1     538.738149802521        0.000000000000      19.798261999832       0.000000000000       0.044279942378
   2     551.283709198289        0.000000000000      20.259302807770       0.000000000000       0.010202925951
   3     561.868173826976        0.000000000000      20.648274711696       0.000000000000       0.002283723743
   4     563.397903080627        0.000000000000      20.704491225347       0.000000000000       0.059443054437
   5     581.900820515202        0.000000000000      21.384460904987       0.000000000000       0.004341104062
   6     588.142051495872        0.000000000000      21.613821914973       0.000000000000       0.022985415235
   7     605.839948563960        0.000000000000      22.264207641561       0.000000000000       0.002655196359
   8     626.548468183689        0.000000000000      23.025231707168       0.000000000000       0.004260067857
   9     639.976645004689        0.000000000000      23.518708107493       0.000000000000       0.003455911458
  10     659.741460916203        0.000000000000      24.245051701200       0.000000000000       0.001575672966
  11     681.888509228761        0.000000000000      25.058940721637       0.000000000000       0.000707120964
  12     700.277330442184        0.000000000000      25.734717442452       0.000000000000       0.000486588641
  13     722.217425682932        0.000000000000      26.541001077715       0.000000000000       0.000167164781
  14     739.219022822455        0.000000000000      27.165798253685       0.000000000000       0.000036633276
  15     757.706193073790        0.000000000000      27.845189235009       0.000000000000       0.000005986088
  16     773.087168790511        0.000000000000      28.410429671693       0.000000000000       0.000000367741
  17    1194.317373960015        0.000000000000      43.890354320144       0.000000000000       0.000015262184
  18    1214.680094351891        0.000000000000      44.638670498413       0.000000000000       0.000001205155
  19    1223.454492231716        0.000000000000      44.961123675674       0.000000000000       0.000012085125
  20    1241.841726961498        0.000000000000      45.636842094290       0.000000000000       0.000023959195
  21    1253.606298774915        0.000000000000      46.069182137710       0.000000000000       0.000057458956
  22    1270.197719739049        0.000000000000      46.678905617137       0.000000000000       0.000016104731
  23    1290.984448411214        0.000000000000      47.442803812432       0.000000000000       0.000029300209
  24    1303.675173810143        0.000000000000      47.909179372633       0.000000000000       0.000007721706
  25    1317.463406585241        0.000000000000      48.415887585326       0.000000000000       0.000003713206
  26    1341.598941212785        0.000000000000      49.302852130602       0.000000000000       0.000001185733
  27    1356.153389062964        0.000000000000      49.837718228179       0.000000000000       0.000002663158
  28    1374.614768178804        0.000000000000      50.516161402748       0.000000000000       0.000000285635
  29    1380.712632758485        0.000000000000      50.740253794631       0.000000000000       0.000000003132
  30    1387.666652952750        0.000000000000      50.995809325289       0.000000000000       0.000002145103
