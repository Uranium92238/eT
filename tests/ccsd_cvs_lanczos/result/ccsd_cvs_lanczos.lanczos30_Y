Nbr.      energy (Re) [eV]      energy (Im) [eV]     energy (Re) [a.u]    energy (Im) [a.u.]   Osc. strength
   1     537.206344420000        0.000000000000      19.741969189852       0.000000000000       0.024680371461
   2     551.768428205174        0.000000000000      20.277115902866       0.000000000000       0.020517273232
   3     560.338580725065        0.000000000000      20.592063201530       0.000000000000       0.058339304963
   4     571.189233412740        0.000000000000      20.990817336277       0.000000000000       0.004569319519
   5     581.285767821930        0.000000000000      21.361858135219       0.000000000000       0.005647195993
   6     590.001845086254        0.000000000000      21.682168069374       0.000000000000       0.014673665588
   7     609.680276159248        0.000000000000      22.405337078793       0.000000000000       0.002631253545
   8     625.526216876172        0.000000000000      22.987664664211       0.000000000000       0.005518224301
   9     644.004478936878        0.000000000000      23.666728275566       0.000000000000       0.001779598083
  10     662.841111734154        0.000000000000      24.358961768687       0.000000000000       0.001059457302
  11     685.319695237011        0.000000000000      25.185034482746       0.000000000000       0.000812865961
  12     704.299030467763        0.000000000000      25.882512193613       0.000000000000       0.000276006443
  13     723.401279248868        0.000000000000      26.584506894179       0.000000000000       0.000138457030
  14     741.874260114204        0.000000000000      27.263376425182       0.000000000000       0.000021234572
  15     759.147374252907        0.000000000000      27.898151666914       0.000000000000       0.000010439711
  16     779.206119259136        0.000000000000      28.635296955709       0.000000000000       0.000000113582
  17    1192.079651172789        0.000000000000      43.808119523814       0.000000000000       0.000001236223
  18    1196.560584571802        0.000000000000      43.972790790309       0.000000000000       0.000008211388
  19    1218.434766981180        0.000000000000      44.776652173676       0.000000000000       0.000014744139
  20    1236.167315157552        0.000000000000      45.428311305017       0.000000000000       0.000042756741
  21    1255.488052750601        0.000000000000      46.138335321392       0.000000000000       0.000032993980
  22    1268.942258188736        0.000000000000      46.632768255762       0.000000000000       0.000016203221
  23    1290.565692383026        0.000000000000      47.427414812111       0.000000000000       0.000026242779
  24    1301.088219370830        0.000000000000      47.814110549700       0.000000000000       0.000005766274
  25    1322.390405362257        0.000000000000      48.596951452246       0.000000000000       0.000004210984
  26    1343.607946001564        0.000000000000      49.376681695450       0.000000000000       0.000000501276
  27    1356.542149018706        0.000000000000      49.852004893160       0.000000000000       0.000002338029
  28    1368.893840940248        0.000000000000      50.305921202769       0.000000000000       0.000000185219
  29    1381.514980873427        0.000000000000      50.769739544249       0.000000000000       0.000000160851
  30    1392.882215995310        0.000000000000      51.187477733459       0.000000000000       0.000001198060
