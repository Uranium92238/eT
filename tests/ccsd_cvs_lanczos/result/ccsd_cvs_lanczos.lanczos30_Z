Nbr.      energy (Re) [eV]      energy (Im) [eV]     energy (Re) [a.u]    energy (Im) [a.u.]   Osc. strength
   1     537.210744264036        0.000000000000      19.742130881139       0.000000000000       0.000006434633
   2     551.321083099601        0.000000000000      20.260676273321       0.000000000000       0.000007808350
   3     559.290959398793        0.000000000000      20.553563827573       0.000000000000       0.095054116581
   4     569.934758256469        0.000000000000      20.944716224215       0.000000000000       0.000385006859
   5     578.986221283585        0.000000000000      21.277351357922       0.000000000000       0.005316497696
   6     592.213369664515        0.000000000000      21.763440099275       0.000000000000       0.001761270388
   7     608.711878285283        0.000000000000      22.369749113033       0.000000000000       0.002654029213
   8     620.322784493591        0.000000000000      22.796442049577       0.000000000000       0.004004551440
   9     641.831986629237        0.000000000000      23.586890655165       0.000000000000       0.001655277107
  10     656.660880626539        0.000000000000      24.131842462707       0.000000000000       0.000703681550
  11     680.236989280231        0.000000000000      24.998248482465       0.000000000000       0.000334815697
  12     698.341442137583        0.000000000000      25.663574858859       0.000000000000       0.000204555299
  13     717.903386691171        0.000000000000      26.382463067612       0.000000000000       0.000119630476
  14     735.365946024190        0.000000000000      27.024200291882       0.000000000000       0.000047266241
  15     746.364215574433        0.000000000000      27.428379246315       0.000000000000       0.000019362681
  16     759.197433684873        0.000000000000      27.899991317123       0.000000000000       0.000004566796
  17    1204.549246803042        0.000000000000      44.266368714836       0.000000000000       0.000000006953
  18    1209.923474471847        0.000000000000      44.463867940522       0.000000000000       0.000006854315
  19    1230.651805207185        0.000000000000      45.225620051205       0.000000000000       0.000026528959
  20    1243.426042877545        0.000000000000      45.695064630800       0.000000000000       0.000018123248
  21    1255.882938656629        0.000000000000      46.152847110896       0.000000000000       0.000026241136
  22    1269.136166881074        0.000000000000      46.639894268828       0.000000000000       0.000017151735
  23    1291.609775319774        0.000000000000      47.465784152651       0.000000000000       0.000003179274
  24    1303.966855207026        0.000000000000      47.919898466349       0.000000000000       0.000006433646
  25    1329.460884176188        0.000000000000      48.856786758273       0.000000000000       0.000000334971
  26    1341.941725662310        0.000000000000      49.315449226879       0.000000000000       0.000001255634
  27    1364.088336960725        0.000000000000      50.129322187342       0.000000000000       0.000000183557
  28    1370.725204435424        0.000000000000      50.373222570433       0.000000000000       0.000000244390
  29    1386.623752894992        0.000000000000      50.957483454751       0.000000000000       0.000000000134
  30    1396.156832953601        0.000000000000      51.307817688061       0.000000000000       0.000000000033
