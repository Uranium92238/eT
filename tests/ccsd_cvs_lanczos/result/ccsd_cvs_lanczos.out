


                     eT - an electronic structure program

  ------------------------------------------------------------------------
   Author list in alphabetical order:
  ------------------------------------------------------------------------
   J. H. Andersen, A. Balbi, S. Coriani, S. D. Folkestad, T. Giovannini, 
   L. Goletto, T. S. Haugland, A. Hutcheson, I-M. Høyvik, E. F. Kjønstad, 
   H. Koch, T. Moitra, R. H. Myhre, A. C. Paul, M. Scavino, A. Skeidsvoll, 
   Å. H. Tveten
  ------------------------------------------------------------------------


  :: Input file
  ================

     Note: geometry section is excluded from this print

     system
        name: H2O He
        charge: 0
     end system

     do
       excited state
     end do

     memory
        available: 8
     end memory

     solver cholesky
        threshold: 1.0d-12
     end solver cholesky

     solver scf
        algorithm:          scf-diis
        energy threshold:   1.0d-12
        gradient threshold: 1.0d-12
     end solver scf

     method
        hf
        ccsd
     end method

     solver cc gs
        omega threshold:  1.0d-12
        energy threshold: 1.0d-12
     end solver cc gs

     solver cc multipliers
        algorithm: diis
        threshold: 1.0d-12
     end solver cc multipliers

     solver cc es
        algorithm:          asymmetric lanczos
        chain length:       30
        core excitation:    {1}
     end solver cc es


  Running on 2 OMP thread(s)
  Memory available for calculation: 8.000000 GB


  :: Molecular system specifications
  =====================================

     Name:             h2o he
     Charge:           0
     Multiplicity:     1
     Coordinate units: angstrom

     Pure basis functions:         29
     Cartesian basis functions:    30
     Primitive basis functions:    56

     Nuclear repulsion energy (a.u.):             12.116100574587
     Bohr/angstrom value (CODATA 2010):            0.529177210920

     ==============================================================================
                                     Geometry (angstrom)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1 H      0.866810000000     0.601440000000     5.000000000000        1
        2 H     -0.866810000000     0.601440000000     5.000000000000        2
        3 O      0.000000000000    -0.075790000000     5.000000000000        3
        4 He     0.100000000000    -0.020000000000     7.530000000000        4
     ==============================================================================

     ==============================================================================
                                     Geometry (a.u.)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: cc-pvdz
        1 H      1.638033502034     1.136556880358     9.448630622825        1
        2 H     -1.638033502034     1.136556880358     9.448630622825        2
        3 O      0.000000000000    -0.143222342981     9.448630622825        3
        4 He     0.188972612457    -0.037794522491    14.229637717975        4
     ==============================================================================

   - Cholesky decomposition of electronic repulsion integrals solver
  ---------------------------------------------------------------------

  Performs a Cholesky decomposition of the two-electron electronic repulsion 
  integrals in the atomic orbital basis,

  (ab|cd) = sum_J L_ab^J L_cd^J.

  Once the Cholesky basis has been determined, the vectors L^J are constructed 
  and stored to disk. These may either be used directly, or be transformed 
  to the MO basis for use in post-HF calculations. For more information, 
  see S. D. Folkestad, E. F. Kjønstad and H. Koch, JCP, 150(19), (2019)

  - Cholesky decomposition settings:

     Target threshold is:   0.10E-11
     Span factor:           0.10E-01
     Max qual:                  1000

  - Cholesky decomposition system details:

     Total number of AOs:                    29
     Total number of shell pairs:           120
     Total number of AO pairs:              435

     Significant shell pairs:               118
     Significant AO pairs:                  431

     Construct shell pairs:                 120
     Construct AO pairs:                    435

  Iter.  #Sign. ao pairs / shell pairs   Max diagonal    #Qualified    #Cholesky    Cholesky array size
  -------------------------------------------------------------------------------------------------------
     1               406 /     108       0.47383E+01         147             42             17052
     2               319 /      93       0.47165E-01         234            111             35409
     3               250 /      76       0.46944E-03         178            183             45750
     4               187 /      55       0.38270E-05         145            265             49555
     5                87 /      25       0.38106E-07          78            324             28188
     6                21 /       3       0.37202E-09          43            349              7329
     7                 0 /       0       0.36652E-11           7            354                 0
  -------------------------------------------------------------------------------------------------------

  - Summary of Cholesky decomposition of electronic repulsion integrals:

     Final number of Cholesky vectors: 354

 - Testing the Cholesky decomposition decomposition electronic repulsion integrals:

     Maximal difference between approximate and actual diagonal:              0.7161E-12
     Minimal element of difference between approximate and actual diagonal:  -0.1110E-14

  - Finished decomposing the ERIs.

     Total wall time (sec):              0.10700
     Total cpu time (sec):               0.16937


  :: RHF wavefunction
  ======================

  - Cholesky decomposition of AO overlap to get linearly independent orbitals:

  - Orbital details:

     Number of occupied orbitals:         6
     Number of virtual orbitals:         23
     Number of molecular orbitals:       29
     Number of atomic orbitals:          29


  :: Hartree-Fock engine
  =========================

  Drives the calculation of the Hartree-Fock state.

  This is a RHF ground state calculation.
  The following tasks will be performed:

     1) Generate initial SAD density
     2) Calculation of reference state (SCF-DIIS algorithm)


  1) Generate initial SAD density


  2) Calculation of reference state (SCF-DIIS algorithm)

   - Self-consistent field DIIS Hartree-Fock solver
  ----------------------------------------------------

  A DIIS-accelerated Roothan-Hall self-consistent field solver. A least-square 
  DIIS fit is performed on the previous Fock matrices and associated gradients. 
  Following the Roothan-Hall update of the density, the DIIS-fitted Fock 
  matrix is used to get the next orbital coefficients.

  - Hartree-Fock solver settings:

     DIIS dimension:                         8
     Cumulative Fock threshold:       0.10E+01

     Energy threshold:              0.1000E-11
     Gradient threshold:            0.1000E-11

     Coulomb screening threshold:   0.1000E-17
     Exchange screening threshold:  0.1000E-15
     Fock precision:                0.1000E-35
     Integral cutoff:               0.1000E-17

  - Setting initial AO density to SAD

     Energy of initial guess:               -78.492022836315
     Number of electrons in guess:           12.000000000000

  Iteration       Energy (a.u.)      Max(grad.)    Delta E (a.u.)
  ---------------------------------------------------------------
     1           -78.796606592587     0.9053E-01     0.7880E+02
     2           -78.828675852655     0.7128E-01     0.3207E-01
     3           -78.843428154462     0.8290E-02     0.1475E-01
     4           -78.843809541675     0.2414E-02     0.3814E-03
     5           -78.843850846799     0.3301E-03     0.4131E-04
     6           -78.843851669326     0.5527E-04     0.8225E-06
     7           -78.843851692851     0.5865E-05     0.2353E-07
     8           -78.843851693557     0.2356E-05     0.7053E-09
     9           -78.843851693630     0.2071E-06     0.7373E-10
    10           -78.843851693631     0.1594E-07     0.4263E-12
    11           -78.843851693631     0.3322E-08     0.2842E-13
    12           -78.843851693631     0.1197E-08     0.1421E-13
    13           -78.843851693631     0.4264E-09     0.7105E-13
    14           -78.843851693631     0.1280E-09     0.9948E-13
    15           -78.843851693631     0.1479E-10     0.7105E-13
    16           -78.843851693631     0.2906E-11     0.2842E-13
    17           -78.843851693631     0.6423E-12     0.8527E-13
  ---------------------------------------------------------------
  Convergence criterion met in 17 iterations!

  - Summary of RHF wavefunction energetics (a.u.):

     HOMO-LUMO gap:                  0.645629080251
     Nuclear repulsion energy:      12.116100574587
     Electronic energy:            -90.959952268218
     Total energy:                 -78.843851693631

  - Molecular orbital energies

  -----------------------------------------------------------------------------------
   1 -20.574269195142   9   0.704416610867  17   1.682695299878  25   3.175416260660
   2  -1.277667044967  10   0.746260122915  18   1.804186820053  26   3.209661156313
   3  -0.898849406811  11   1.155862024381  19   1.902641648743  27   3.328173215200
   4  -0.629870222687  12   1.170770887076  20   2.148883457169  28   3.721936473470
   5  -0.541641772852  13   1.267961746552  21   2.200395756970  29   3.985492632586
   6  -0.485872762834  14   1.449847537220  22   2.540321860021
   7   0.159756317417  15   1.463234913437  23   2.541517310547
   8   0.229311906285  16   1.474444394527  24   2.559338707349
  -----------------------------------------------------------------------------------

  - Timings for the RHF ground state calculation

     Total wall time (sec):              0.49800
     Total cpu time (sec):               0.85624


  :: CCSD wavefunction
  =======================

     Bath orbital(s):         False
     Core-valence separation: False

   - Number of orbitals:

     Occupied orbitals:    6
     Virtual orbitals:     23
     Molecular orbitals:   29
     Atomic orbitals:      29

   - Number of ground state amplitudes:

     Single excitation amplitudes:  138
     Double excitation amplitudes:  9591


  :: Excited state coupled cluster engine
  ==========================================

  Calculates the coupled cluster excitation vectors and excitation energies

  This is a CCSD excited state calculation.
  The following tasks will be performed:

     1) Preparation of MO basis and integrals
     2) Calculation of the ground state (diis algorithm)
     3) Calculation of the multipliers (diis algorithm)
     4) Calculation of the excited state (asymmetric lanczos algorithm)


  1) Preparation of MO basis and integrals

  - Settings for integral handling:

     Cholesky vectors in memory: True
     ERI matrix in memory:       True


  2) Calculation of the ground state (diis algorithm)

   - DIIS coupled cluster ground state solver
  ----------------------------------------------

  A DIIS CC ground state amplitude equations solver. It uses an extrapolation 
  of previous quasi-Newton perturbation theory estimates of the next amplitudes. 
  See Helgaker et al., Molecular Electronic Structure Theory, Chapter 
  13.

  - DIIS CC ground state solver settings:

     Omega threshold:           0.10E-11
     Energy threshold:          0.10E-11

     DIIS dimension:                   8
     Max number of iterations:       100

     Storage: disk

  Iteration    Energy (a.u.)        |omega|       Delta E (a.u.)
  ---------------------------------------------------------------
    1           -79.084241931608     0.9391E-01     0.7908E+02
    2           -79.092586666607     0.2720E-01     0.8345E-02
    3           -79.099408028306     0.7507E-02     0.6821E-02
    4           -79.100345871309     0.2095E-02     0.9378E-03
    5           -79.100371860301     0.5154E-03     0.2599E-04
    6           -79.100393801016     0.2313E-03     0.2194E-04
    7           -79.100385611260     0.4933E-04     0.8190E-05
    8           -79.100384217524     0.1180E-04     0.1394E-05
    9           -79.100383621794     0.4135E-05     0.5957E-06
   10           -79.100383427012     0.1779E-05     0.1948E-06
   11           -79.100383466393     0.6705E-06     0.3938E-07
   12           -79.100383474630     0.2989E-06     0.8237E-08
   13           -79.100383487352     0.1028E-06     0.1272E-07
   14           -79.100383481865     0.3056E-07     0.5487E-08
   15           -79.100383481092     0.6508E-08     0.7730E-09
   16           -79.100383481303     0.2112E-08     0.2110E-09
   17           -79.100383481484     0.6588E-09     0.1819E-09
   18           -79.100383481546     0.1594E-09     0.6156E-10
   19           -79.100383481554     0.5326E-10     0.7915E-11
   20           -79.100383481555     0.2008E-10     0.1194E-11
   21           -79.100383481556     0.7516E-11     0.6821E-12
   22           -79.100383481556     0.3071E-11     0.2984E-12
   23           -79.100383481556     0.1336E-11     0.2842E-13
   24           -79.100383481556     0.5367E-12     0.4263E-13
  ---------------------------------------------------------------
  Convergence criterion met in 24 iterations!

  - Ground state summary:

     Final ground state energy (a.u.):   -79.100383481556

     Correlation energy (a.u.):           -0.256531787925

     Largest single amplitudes:
     -----------------------------------
        a       i         t(a,i)
     -----------------------------------
        1      5       -0.014740597523
       14      4        0.009546856221
        7      4       -0.008284826484
       15      5        0.006124828874
        4      5       -0.005606072698
        6      2       -0.005476844297
        2      4       -0.005318591697
       13      5       -0.005269818337
        5      6        0.004933006908
       11      6       -0.003454309398
     ------------------------------------

     Largest double amplitudes:
     --------------------------------------------------
        a      i       b      j         t(ai,bj)
     --------------------------------------------------
        2      4       2      4       -0.047351708919
        5      6       5      6       -0.046240574404
        9      3       9      3       -0.041367012249
        3      4       3      4       -0.036659067517
        6      5       6      5       -0.034554012170
        1      5       1      5       -0.034177347750
       16      3      16      3       -0.032108235347
       17      3      17      3       -0.032052553603
       18      3      18      3       -0.031351828684
        2      4       3      4       -0.029701270698
     --------------------------------------------------

     T1 diagnostic (|T1|/sqrt(N_e)): 0.007226152112

  - Finished solving the CCSD ground state equations

     Total wall time (sec):              0.39800
     Total cpu time (sec):               1.12782


  4) Calculation of the excited state (asymmetric lanczos algorithm)
     Calculating right vectors


  3) Calculation of the multipliers (diis algorithm)

   - DIIS coupled cluster multipliers solver
  ---------------------------------------------

  A DIIS CC multiplier equations solver. It combines a quasi-Newton perturbation 
  theory estimate of the next multipliers, using least square fitting 
  to find an an optimal combination of previous estimates such that the 
  update is minimized.

  See Helgaker et al., Molecular Electronic Structure Theory, Chapter 
  13, for the more details on this algorithm.

  - DIIS CC multipliers solver settings:

     Residual threshold:        0.10E-11

     DIIS dimension:                   8
     Max number of iterations:       100

  Iteration    Norm residual
  ----------------------------
    1          0.2076E+01
    2          0.2758E+00
    3          0.7390E-01
    4          0.1823E-01
    5          0.5144E-02
    6          0.1276E-02
    7          0.4964E-03
    8          0.1188E-03
    9          0.3340E-04
   10          0.1139E-04
   11          0.4270E-05
   12          0.1506E-05
   13          0.7168E-06
   14          0.2642E-06
   15          0.7915E-07
   16          0.2040E-07
   17          0.5933E-08
   18          0.2025E-08
   19          0.5601E-09
   20          0.2374E-09
   21          0.7979E-10
   22          0.2750E-10
   23          0.1089E-10
   24          0.5256E-11
   25          0.2457E-11
   26          0.7653E-12
  ----------------------------
  Convergence criterion met in 26 iterations!

  - DIIS CC multipliers solver summary:

     Largest single amplitudes:
     -----------------------------------
        a       i         r(a,i)
     -----------------------------------
        1      5       -0.022790118896
       14      4        0.015907507091
        7      4       -0.014297181949
        4      5       -0.011712306720
       15      5        0.010007216962
        6      2       -0.009083547621
       13      5       -0.008770531160
        2      4       -0.006954460286
        5      6        0.006820250283
        3      4       -0.006765159552
     ------------------------------------

     Largest double amplitudes:
     --------------------------------------------------
        a      i       b      j         r(ai,bj)
     --------------------------------------------------
        5      6       5      6       -0.092500754790
        2      4       2      4       -0.092085328212
        6      5       5      6        0.089666625590
        2      4       5      6        0.083792278098
        9      3       9      3       -0.082167759498
        2      4       6      5       -0.077164163662
        2      4       1      5       -0.076342716082
        7      4       5      6       -0.071424572153
        3      4       3      4       -0.071282985174
        6      5       6      5       -0.068075967748
     --------------------------------------------------

  - Finished solving the ccsd multipliers equations

     Total wall time (sec):              0.27000
     Total cpu time (sec):               0.46522

   - Asymmetric Lanczos excited state solver
  ---------------------------------------------

  An asymmetric Lanczos solver builds a reduced space tridiagonal representation 
  of the CC Jacobian of dimension defined by the chain length. Diagonalization 
  of this representation gives the eigenvalues and eigenvectors.

  A complete description of the algorithm can be found in  S.Coriani et 
  al., J. Chem. Theory Comput. 2012, 8, 5, 1616-1628.

  - Settings for coupled cluster excited state solver (asymmetric lanczos):

     Chain length:     30
     Biorthonormalization procedure: asymmetric

  - Summary of the asymmetric Lanczos solver for excited states

     Printing the 10 lowest excited states for each Cartesian component 
     of the electric dipole moment

     Component: X

     State.      energy [a.u]         energy [eV]         Osc. strength
     ----------------------------------------------------------------------
        1       19.798261999832     538.738149802520      0.044279942378
        2       20.259302807770     551.283709198289      0.010202925951
        3       20.648274711769     561.868173828963      0.002283723749
        4       20.704491225351     563.397903080720      0.059443054431
        5       21.384460905007     581.900820515739      0.004341104063
        6       21.613821914978     588.142051495997      0.022985415235
        7       22.264207641571     605.839948564229      0.002655196359
        8       23.025231707177     626.548468183926      0.004260067857
        9       23.518708107503     639.976645004957      0.003455911458
       10       24.245051701207     659.741460916389      0.001575672966
     ----------------------------------------------------------------------
     For full spectrum see file: h2o he_30_X

     Component: Y

     State.      energy [a.u]         energy [eV]         Osc. strength
     ----------------------------------------------------------------------
        1       19.741969189852     537.206344420000      0.024680371461
        2       20.277115902866     551.768428205173      0.020517273232
        3       20.592063201530     560.338580725066      0.058339304963
        4       20.990817336279     571.189233412807      0.004569319519
        5       21.361858135227     581.285767822144      0.005647195993
        6       21.682168069377     590.001845086322      0.014673665588
        7       22.405337078800     609.680276159434      0.002631253545
        8       22.987664664215     625.526216876267      0.005518224301
        9       23.666728275573     644.004478937066      0.001779598083
       10       24.358961768693     662.841111734315      0.001059457302
     ----------------------------------------------------------------------
     For full spectrum see file: h2o he_30_Y

     Component: Z

     State.      energy [a.u]         energy [eV]         Osc. strength
     ----------------------------------------------------------------------
        1       19.742130881139     537.210744264035      0.000006434633
        2       20.260676273321     551.321083099604      0.000007808350
        3       20.553563827573     559.290959398792      0.095054116581
        4       20.944716224216     569.934758256494      0.000385006859
        5       21.277351357922     578.986221283589      0.005316497696
        6       21.763440099276     592.213369664535      0.001761270388
        7       22.369749113034     608.711878285309      0.002654029213
        8       22.796442049577     620.322784493605      0.004004551440
        9       23.586890655166     641.831986629263      0.001655277107
       10       24.131842462708     656.660880626584      0.000703681550
     ----------------------------------------------------------------------
     For full spectrum see file: h2o he_30_Z

  - Finished solving the CCSD excited state equations.

     Total wall time (sec):              2.22800
     Total cpu time (sec):               3.78721

  - Timings for the CCSD excited state calculation

     Total wall time (sec):              2.90300
     Total cpu time (sec):               5.38943

  eT terminated successfully!
